.github/workflows/wheel_tests_and_release.yml

```
@@ -0,0 +1,213 @@
+name: Build Wheels and Release
+
+on:
+  push:
+    branches:
+      - cibuildwheel
+      - master
```

```suggestion
      - master
```
might just remove cibuildwheel once you are done testing since we probably don't want to keep a cibuildwheel branch on the main repo. we can always still push a tag starting with `buildwheels` to test things out there.